presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -772,6 +772,7 @@ public void createDatabase(Database database)
                     .stopOnIllegalExceptions()
                     .run("createDatabase", stats.getCreateDatabase().wrap(() -> {
                         try (ThriftMetastoreClient client = createMetastoreClient()) {
+                            client.setUGI(database.getOwnerName());
```

Comment:
- what should we do when `hive.metastore.username` is also set?
- we should have a test for this, that would run in plain and Kerberized envs

Reply:
I feel setting the same user for metastore and ugi is straightforward in such case because current CREATE TABLE statement does so.

Let me add those tests.